<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartWifiWidget</class>
 <widget class="QWidget" name="SmartWifiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartWifi</string>
  </property>
  <widget class="QLabel" name="connect_flag">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="connect_note">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Server is disconnected</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable SmartWifi</string>
   </property>
   <property name="group" stdset="0">
    <string>big_btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="whitelist_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>White List</string>
   </property>
   <property name="group" stdset="0">
    <string>big_btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="purchases_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchases</string>
   </property>
   <property name="group" stdset="0">
    <string>big_btn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="account_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Acount Status</string>
   </property>
   <property name="group" stdset="0">
    <string>big_btn</string>
   </property>
  </widget>
  <widget class="QWidget" name="top_bar" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QLabel" name="server_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>6</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Name</string>
    </property>
    <property name="group" stdset="0">
     <string>top</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_img">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="group" stdset="0">
     <string>top</string>
    </property>
   </widget>
   <widget class="QLabel" name="server_img">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>6</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="group" stdset="0">
     <string>top</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_img">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>5</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="group" stdset="0">
     <string>top</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>6</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>|  status</string>
    </property>
    <property name="group" stdset="0">
     <string>top</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>6</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>|  speed</string>
    </property>
    <property name="group" stdset="0">
     <string>top</string>
    </property>
   </widget>
   <widget class="QLabel" name="server_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>7</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="group" stdset="0">
     <string>top</string>
    </property>
   </widget>
   <zorder>speed_img</zorder>
   <zorder>status_img</zorder>
   <zorder>status_label</zorder>
   <zorder>speed_label</zorder>
   <zorder>server_label</zorder>
   <zorder>server_img</zorder>
   <zorder>server_name</zorder>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <widget class="QWidget" name="flow_page">
    <widget class="QLabel" name="flow_account_img">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>5</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="group" stdset="0">
      <string>account_information_img</string>
     </property>
    </widget>
    <widget class="QLabel" name="fow_account_text">
     <property name="geometry">
      <rect>
       <x>263</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Account Information</string>
     </property>
     <property name="group" stdset="0">
      <string>account_information_text</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Service Type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Flow Charged</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Residul Flow;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>100 MB</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>321</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>166</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>line</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="flow_note">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Note:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="group" stdset="0">
      <string>account_note</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>41</width>
     <height>331</height>
    </rect>
   </property>
   <property name="text">
    <string>line</string>
   </property>
  </widget>
  <zorder>top_bar</zorder>
  <zorder>connect_flag</zorder>
  <zorder>connect_note</zorder>
  <zorder>connect_btn</zorder>
  <zorder>whitelist_btn</zorder>
  <zorder>purchases_btn</zorder>
  <zorder>account_btn</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label_8</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
